packages/KInput/CHANGELOG.md

```diff
@@ -3,60 +3,10 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## [6.1.2](https://github.com/Kong/kongponents/compare/@kongponents/kinput@6.1.1...@kongponents/kinput@6.1.2) (2021-11-17)
-
-**Note:** Version bump only for package @kongponents/kinput
-
-
-
-
-
-## [6.1.1](https://github.com/Kong/kongponents/compare/@kongponents/kinput@6.0.1-alpha.3...@kongponents/kinput@6.1.1) (2021-11-17)
-
-**Note:** Version bump only for package @kongponents/kinput
-
-
-
-
-
-## [6.0.1-alpha.3](https://github.com/Kong/kongponents/compare/@kongponents/kinput@6.0.1-alpha.2...@kongponents/kinput@6.0.1-alpha.3) (2021-11-17)
-
-**Note:** Version bump only for package @kongponents/kinput
-
-
-
-
-
-## 6.0.1-alpha.2 (2021-11-17)
-
-**Note:** Version bump only for package @kongponents/kinput
-
-
-
-
-
-## 6.0.1-alpha.1 (2021-11-16)
-
-**Note:** Version bump only for package @kongponents/kinput
-
-
-
-
-
-## 6.0.1-alpha.0 (2021-11-16)
-
-**Note:** Version bump only for package @kongponents/kinput
-
-
-
-
-
-## 5.0.9-alpha.0 (2021-11-16)
-
-**Note:** Version bump only for package @kongponents/kinput
-
-
+## [6.1.2](https://github.com/Kong/kongponents/compare/@kongponents/kinput@5.0.8...@kongponents/kinput@6.1.2) (2021-11-17)
 
+### Features
+  * `size` prop added
 
```

Comment:
Mention fix for attributes now being properly applied to only the child element instead of the wrapper? If yes, applies to:

* `KInput`
* `KCheckbox`
* `KInputSwitch`
* `KRadio`
* `KSelect`
* `KTextArea`

Reply:
Done